LongWarOfTheChosen/Src/LW_FactionBalance/Classes/X2DownloadableContentInfo_LW_FactionBalance.uc

```diff
@@ -30,6 +30,7 @@ static event OnPostTemplatesCreated()
 	UpdateRemoteStart();
 	//Combat Intelligenve Covert Action
 	UpdateCICA();
+  AllowTwoSoldiersFromEachFaction();

```

Comment:
You're triggering my OCD again 😆 Would you mind using the same indent as the previous line.